packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -100,13 +101,17 @@ export class ModelInstanceDocument<T = Record<string, any>> extends Stream {
 
   private _isReadOnly = false
 
-  get content(): T {
+  override get content(): T {
     return super.content
   }
 
   get metadata(): ModelInstanceDocumentMetadata {
     const metadata = this.state$.value.metadata
-    return { controller: metadata.controllers[0], model: metadata.model }
+    const controller = metadata.controllers[0]
+    if (!controller) throw new Error(`Controller is not present`)
+    const model = metadata.model
+    if (!model) throw new Error(`Model is not present`)
```

Comment:
same for model, both of these should be checked by the model-instance-document-handler as part of loading the stream in the first place.  If it gets far enough to have a ModelInstanceDocument instance, this should already be guaranteed to be set